src/polyswarm_api/__init__.py

```diff
@@ -81,6 +81,16 @@ def __init__(self, key, uri='https://api.polyswarm.network/v1', get_limit=10,
                 logger.warning('To update, run: pip install -U polyswarm-api')
                 logger.warning('Please upgrade or certain features may not work properly.')
 
+    def _get_hash_type(self, value):
```

Comment:
this function is also in the `__main__.py`. It should only be in one place

Reply:
ok, created `utils.py` for it